__init__.py

```diff
@@ -9,10 +9,14 @@
 
 from albert import *
 
-__title__ = "Jetbrains IDE Projects"
-__version__ = "0.4.5"
-__triggers__ = "jb "
-__authors__ = "Markus Richter, Thomas Queste"
+md_iid = "0.5"
+md_version = "1.0"
```

Comment:
maybe more like 0.5? I think I would do 1.0 if the app icons work again :)

Reply:
:+1: set to `0.5`